modules/assignment/src/main/resources/assignment.properties

```diff
@@ -27,6 +27,14 @@ http.proxy.host:${http.proxy.host}
 http.proxy.port:${http.proxy.port}
 ruleCache.executor.pool.size:5
 
+#Specify the number of messages to be drained on the event of queue full (executor.max.queue.size).
+#Default value is 10_000
+drain.queue.by=10000
```

Comment:
@abhishekjain88 @iizrailevsky Yes, for consistency, I have added these placeholder configs in wasabi-open source which could be used for queue draining functionality in future...